config/default/kustomization.yaml

```diff
@@ -0,0 +1,72 @@
+# Adds namespace to all resources.
+namespace: kubit
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: kubit-
```

Comment:
do we need a name prefix?

Reply:
ah yeah we call the deployment `controller-manager`, so I guess it will be called `kubit-controller-manager`

Reply:
Maybe not, but it seems like an easy thing to have right?

Everything from the projects looks like `kubit-SOMETHING` to keep it together, similar to how everything Nginx related comes in with `nginx-SOMETHING`.

We can take it out if you think its not required, I don't think its a huge deal though.